### DISCUSSION TITLE
use this template is missing and Im a newb, can someone help me ?

### DISCUSSION THREAD

**Discussion by paleskincodes** [community/community#138690] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

"use this template" is  missing from my github dash, which since Im a newb, its hard for me to use others' repos as templates

can someone help me

**Comment by gowthamprasath14** [comment]
good

**Comment by techy4shri** [comment]
Can you elaborate, where it is missing from? If you mean the option isn't available for your own repo then it can be changed in the settings of the concerned repository. 
Follow these steps: 

1. Go to your repository
2. Click on setttings
3. Check the box for Template Repository under General.

If you mean the option is not available for a repository not owned by you, then it is just a permission setting. Basically the owner of that repository might not have allowed the template to be available for others.
You can only use templates of repositories which allow it, otherwise you cannot.

Hope this helps! Please mark this as answered if it did or feel free to ask follow up questions in replies! 

Edit: That repo doesn't have the option to use it as template because the owner has not allowed it. Although there is a link in their description which points to a repo and it has the template option.
Repo with template permission: https://github.com/zizifn/edgetunnel

So maybe they want you to template that repo instead.

**Reply by paleskincodes** [reply]
> Can you elaborate, where it is missing from? If you mean the option isn't available for your own repo then it can be changed in the settings of the concerned repository. Follow these steps:
> 
>     1. Go to your repository
> 
>     2. Click on setttings
> 
>     3. Check the box for Template Repository under General.
> 
> 
> If you mean the option is not available for a repository not owned by you, then it is just a permission setting. Basically the owner of that repository might not have allowed the template to be available for others. You can only use templates of repositories which allow it, otherwise you cannot.
> 
> Hope this helps! Please mark this as answered if it did or feel free to ask follow up questions in replies!

![missing template git](https://github.com/user-attachments/assets/4a38d618-1049-48f6-97d1-13ce6ebf6412)

no not my repo, but other's repo, which they told ppl to use as template

**Reply by techy4shri** [reply]
I see. That repo doesn't have the option to use it as template because the owner has not allowed it. Although there is a link in their description which points to a repo and it has the template option. 
![image](https://github.com/user-attachments/assets/25bbb1fb-ca9f-44dd-9a79-9bf272a1fb71)

https://github.com/zizifn/edgetunnel 

So maybe they want you to template that repo instead. Confirm with them once!

**Reply by techy4shri** [reply]
Updated my reply with some new info, check it and do mark as answer if it helped you!
